wordpress-mobile/WordPress-iOS
# Change navigation header

Fixes #13835

### To test

* Open the Nudges view
* Tap Visibility or Tags
* Check the Navigation Header, it should display only the title

Ps.: Test Light and Dark Mode.
